ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -379,6 +379,9 @@ function build_upi_cluster() {
 
     echo "Copying the terraform.tfstate"
     cp "${IBMCLOUD_HOME}"/ocp4-upi-powervs/terraform.tfstate "${SHARED_DIR}"/terraform.tfstate
+    if [ ! -f "/root/.kube/config" ]; then
```

Comment:
failure was in an earlier step... still debugging in this PR / rehearsal
root copy failure isn't on this system - it's on the bastion.